client/src/Components/Pages/Signup/index.js

```diff
@@ -173,18 +173,18 @@ const CustomCheckbox = ({ field, ...props }) => (
 
 export default class Signup extends Component {
   state = {
-    isWorker: "yes",
+    isWorker: null,
     orgType: "agency",
     ismodalVisible: false,
     newTrade: "",
     error: "",
-    errors: {},
     isPopupVisible: false,
     data: null,
   };
 
-  handleSubmit = (values, { setSubmitting }) => {
+  handleSubmit = (_values, { setSubmitting }) => {
```

Comment:
can you explain the _ again please? Sorry completely forgot what this means again